vendor/modules.txt

```diff
@@ -47,7 +48,7 @@ github.com/blang/semver
 github.com/certifi/gocertifi
 # github.com/containerd/cgroups v0.0.0-20190919134610-bf292b21730f
 github.com/containerd/cgroups/stats/v1
-# github.com/containerd/containerd v1.3.0 => github.com/containerd/containerd v1.3.0
+# github.com/containerd/containerd v1.3.2 => github.com/containerd/containerd v1.3.0
```

Comment:
@otaviof so it looks like a transitive dep now uses `containerd/containerd v1.3.2`. We have a replace statement to pin containerd to v1.3.0 - please update the replace to use the new tag.

Reply:
Just pushed the changes to make sure `containerd/containerd` version `1.3.2` in place. Please consider.